### ISSUE TITLE
Selection doesn't work in the content editable accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#174078]
1. Set `terminal.integrated.accessibleBufferContentEditable: on`
2. focus a terminal
3. run some commands
4. `Shift+Tab` and try to select 🐛 

fyi @Tyriar

**Comment by meganrogge** [comment]
probably because i'm setting the selection so that the cursor is at the end

**Comment by Tyriar** [comment]
The setting of selection is just the initial state to position the cursor though, you should still be able to use shift+arrow to select something different/

**Comment by meganrogge** [comment]
Cmd+A doesn't select all for me but I can use cmd+A 🤔

**Comment by Tyriar** [comment]
cmd+a is likely running the select all in terminal command, you could check if the accessible buffer is up and select all in that instead (or not run the command and pass the event through?)

**Comment by Tyriar** [comment]
Or have a new context key for when it's up?

**Comment by meganrogge** [comment]
much clearer to see what's going on when I have an editor open - the terminal (ie terminal text area) is explicitly not focused when the accessible buffer is. the terminal's select all has `when: TerminalContextKeys.focus`
https://user-images.githubusercontent.com/29464607/218154386-e75318d5-1029-4bb6-aafa-be6c8d2c9071.mov

**Comment by meganrogge** [comment]
So seems like yeah new context key
